.github/workflows/ci.yml

```diff
@@ -37,8 +37,8 @@ jobs:
       run: ./config --strict-warnings enable-fips && perl configdata.pm --dump
     - name: make build_generated
       run: make -s build_generated
-    - name: make doc-nits
-      run: make doc-nits
+    - name: make doc-nits cmd-nits
+      run: make doc-nits cmd-nits
```

Comment:
cmd-nits requires the openssl command to be built.  might be worth adding it as a task to an existing build-programs target.

Reply:
That would be easier.  From clean `make cmd-nits` isn't working.

Reply:
Moved.